crates/orchestrator/src/telemetry.rs

```diff
@@ -13,33 +12,26 @@ use tracing::Level;
 use tracing_opentelemetry::OpenTelemetryLayer;
 use tracing_subscriber::layer::SubscriberExt as _;
 use tracing_subscriber::util::SubscriberInitExt as _;
-use utils::env_utils::get_env_var_or_panic;
+use utils::env_utils::{get_env_var_optional, get_env_var_or_panic};
 
 lazy_static! {
     #[derive(Debug)]
     pub static ref OTEL_SERVICE_NAME: String = get_env_var_or_panic("OTEL_SERVICE_NAME");
 }
 
-pub fn setup_analytics() -> Option<SdkMeterProvider> {
-    let otel_endpoint = get_env_var_or_panic("OTEL_COLLECTOR_ENDPOINT");
-    let tracing_level = Level::from_str(&get_env_var_or_panic("TRACING_LEVEL"))
-        .expect("Could not obtain tracing level from environment variable.");
-
-    // guard clause if otel is disabled
-    if otel_endpoint.is_empty() {
-        return None;
-    }
+pub fn setup_analytics(level: Level) -> Option<SdkMeterProvider> {
+    get_env_var_optional("OTEL_COLLECTOR_ENDPOINT").unwrap_or(None).map(|otel_endpoint| {
```

Comment:
it's already optional, what does `unwrap_or(None)` do?

Reply:
Fixed.